src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/IndexManagementIndices.kt

```diff
@@ -35,6 +35,8 @@ import org.elasticsearch.cluster.service.ClusterService
 import org.elasticsearch.common.settings.Settings
 import org.elasticsearch.common.xcontent.XContentType
 
+private val log = LogManager.getLogger(IndexManagementIndices::class.java)
```

Comment:
Not used?

Reply:
removed, we have a logger in this file...